docs/pages/operator/cluster-api-provider.mdx

```diff
@@ -0,0 +1,74 @@
+```
+
+## Installing the CAPI stack with the vcluster provider
+Switch your kubectl context to the cluster where you wish to install the CAPI stack with the vcluster provider and run the following command:
+```shell
+clusterctl init --infrastructure vcluster
+```
+
+The `"Your management cluster has been initialized successfully!"` message should eventually appear in the output.
+
+## Creating a vcluster
+We will use the clusterctl CLI tool to generate the Kubernetes resource definitions needed for vcluster creation by the CAPI provider. The output is in YAML format, and it can be saved into a file to be used later, or it can be forwarded in the command line into the kubectl command to immediately apply the resources in the cluster.  
+
+A vcluster instance created via vcluster provider is configured using clusterctl parameters and environment variables - CHART_NAME, CHART_REPO, CHART_VERSION, VCLUSTER_HOST, VCLUSTER_PORT, and HELM_VALUES.
+
+We will define some configuration elements(CLUSTER_NAME and CLUSTER_NAMESPACE) as environment variables for easier reuse across the commands. These will be used together with some of the environment variables mentioned above to produce a manifest defining your vcluster instance and apply it in the host cluster:
+```
+export CLUSTER_NAME=vcluster
+export CLUSTER_NAMESPACE=vcluster
+export KUBERNETES_VERSION=1.23.0
+export HELM_VALUES=""
+kubectl create namespace ${CLUSTER_NAMESPACE}
+clusterctl generate cluster ${CLUSTER_NAME} \
+    --infrastructure vcluster \
+    --target-namespace ${CLUSTER_NAMESPACE} | kubectl apply -f -
+```
+
+:::tip
+You can populate the HELM_VALUES variable from a `values.yaml` file using this command:
+```export HELM_VALUES=$(cat values.yaml | sed -z 's/\n/\\n/g')```
+:::
+
+Next, we need to wait until vcluster custom resource reports ready status:
+```shell
+kubectl wait --for=condition=ready vcluster -n $CLUSTER_NAMESPACE $CLUSTER_NAME 
+```
+The cluster is ready to be used once the command above exists.
+
+### Connecting to your vcluster
+There are multiple methods for connecting to your vcluster instance, and they are described on the [Accessing vcluster page](./external-access). If the documentation instructs you to update your values.yaml file, you will need to update the HELM_VALUES environment variable, execute the clusterctl generate cluster again, and apply the updated output to you cluster. Alternatively, you can edit the `.spec.helmRelease.values` field of the vcluster custom resource using `kubectl edit vcluster -n $CLUSTER_NAMESPACE $CLUSTER_NAME ` command, but be mindful of the formatting of the value, the string should be a valid YAML file, with the expected indentation for the various helm values. The vcluster provider is watching for the changes made to vcluster custom resource, and it will automatically update the vcluster instance when changes are detected.
```

Comment:
I'd add a line break between "If the documentation instructs you to update your values.yaml file, you will need to update the HELM_VALUES environment variable, execute the clusterctl generate cluster again, and apply the updated output to you cluster." and "Alternatively, you can edit the `.spec.helmRelease.values` field of the vcluster custom resource using `kubectl edit vcluster -n $CLUSTER_NAMESPACE $CLUSTER_NAME ` command, but be mindful of the formatting of the value, the string should be a valid YAML file, with the expected indentation for the various helm values." Together it gets a bit wordy. I'd also split the "Alternatively" sentence.

"There are multiple methods for connecting to your vcluster instance, and they are described on the [Accessing vcluster page](./external-access). If the documentation instructs you to update your values.yaml file, you will need to update the HELM_VALUES environment variable, execute the clusterctl generate cluster again, and apply the updated output to you cluster. 

Alternatively, you can edit the `.spec.helmRelease.values` field of the vcluster custom resource using `kubectl edit vcluster -n $CLUSTER_NAMESPACE $CLUSTER_NAME ` command. Be mindful of the formatting of the value: the string should be a valid YAML file, with the expected indentation for the various helm values. The vcluster provider is watching for the changes made to vcluster custom resources, and it will automatically update the vcluster instance when changes are detected."